tools/scripts/xabuild

```diff
@@ -52,11 +52,24 @@ if [ -z "$CONFIGURATION" ]; then
 		esac
 
 		if [ -n "$CONFIGURATION" ]; then
+			for word in $CONFIGURATION; do
+				CONFIGURATION=$word
+				break
+			done
+
 			break
 		fi
 	done
 fi
 
+if [[ "$MSBUILD" == "msbuild" ]] ; then
```

Comment:
This "block" should be moved to [~line 84](https://github.com/xamarin/xamarin-android/blob/eaf9919/tools/scripts/xabuild#L84), *after* `$prefix` is set. `$prefix` contains the `$CONFIGURATION` to use.